pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -108,16 +110,25 @@ const DepositDates = ({
           count: store.depositDates.depositDatesCount || '',
         })}
       </p>
-      <Button
-        variant="contained"
-        onClick={() => store.depositDates.exportCsv()}
-        disabled={
-          store.depositDates.isExportInProgress ||
-          store.depositDates.isExportDisabled
-        }
+      <span
+        className={classNames.use({
+          [styles.exportButtonDisabled]: store.depositDates.isExportDisabled,
+        })}
```

Comment:
Is there any problem with passing a class name into the _Button_ directly?

Reply:
Yes, the cursor: not allowed doesn't work. pointer-events takes precedence over it.